includes/core-changes/cryptography/3.0/cryptography-cryptostream-dispose-final-block-write.md

```diff
@@ -0,0 +1,38 @@
+### System.Security.Cryptography.CryptoStream.Dispose is transforming final block only when writing
+
+The <xref:System.Security.Cryptography.CryptoStream.Dispose?displayProperty=fullName> which is used to finish crypto stream operations was previously throwing an exception when used with some crypto transforms (i.e. AES with padding) and reading was not complete. This scenario can occur i.e. in network scenarios when operation is cancelled and causing users to have to catch the exception.
```

Comment:
```suggestion
The <xref:System.Security.Cryptography.CryptoStream.Dispose?displayProperty=fullName> method, which is used to finish CryptoStream operations, was previously throwing an exception when used with some crypto transforms (i.e. AES with padding) and reading was not complete. This scenario can occur i.e. in network scenarios when operation is cancelled and causing users to have to catch the exception.
```

(inserted "method" between "Dispose" and "which"; moved which ... operations to an aside, changed "crypto stream" to "CryptoStream")